src/components/PageLanding.js

```diff
@@ -106,7 +106,7 @@ class PageLanding extends Component {
           </Col>
         </Row>
         <Row className="show-grid neighborhood-row">
-          <Col md={textColWidth}>
+          <Col xs={textColWidth}>
```

Comment:
can we push/pull these columns so the icons all line up on the same side for small screens? I think the desktop view can stay as existing.
https://getbootstrap.com/docs/3.4/css/#grid-column-ordering

original:

![image](https://user-images.githubusercontent.com/5168796/53515767-a2513780-3a7f-11e9-893c-77ebe2abce73.png)

proposed:
![image](https://user-images.githubusercontent.com/5168796/53515908-e80e0000-3a7f-11e9-85ca-747e8d06210c.png)